types/jasmine/jasmine-tests.ts

```diff
@@ -1586,7 +1586,8 @@ describe("Custom async matcher: 'toBeEight'", () => {
 describe('better typed spys', () => {
     describe('a typed spy', () => {
         const spy = jasmine.createSpy('spy', (num: number, str: string) => {
```

Comment:
OOC you could have also change this to

```ts
const spy = jasmine.createSpy('spy', (num: number, str: string): string => {
```

right?

Reply:
Yep, that's a better fix. I'll do that instead.